Src/Coravel/Scheduling/Schedule/Scheduler.cs

```diff
@@ -73,6 +73,13 @@ public IScheduleInterval ScheduleAsync(Func<Task> asyncTaskToSchedule)
             return scheduled;
         }
 
+        public IScheduleInterval ScheduleWithParams(Type invocableType, params object[ ] parameters)
+        {
+            ScheduledEvent scheduled = ScheduledEvent.WithInvocableAndParams(invocableType, this._scopeFactory, parameters);
+            this._tasks.TryAdd (Guid.NewGuid ().ToString (), new ScheduledTask (this._currentWorkerName, scheduled));
```

Comment:
Minor code style: Could you remove the spaces before each `(` please? For example, `ToString ()` would become `ToString()`. 👍

Reply:
Resolved 😄.